### ISSUE TITLE
[Screen Readers- LS E-mail Template]: NVDA is not announcing the descriptive information on the Microsoft logo image

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13935]
**Screenshot/Video**
[attachments.zip](https://github.com/nvaccess/nvda/files/9158293/attachments.zip)

### Steps to reproduce:
Step 1: Open the above-mentioned template [New Compressed (zipped) Folder.zip](https://github.com/nvaccess/nvda/files/9158263/New.Compressed.zipped.Folder.zip)
Step 2: Navigate to Microsoft logo image
Step 3: Verify whether NVDA is not announcing the descriptive information on the Microsoft logo image or not.

### Actual behavior:
Users who rely on Assistive Technology such as screen readers are getting impacted. Screen reader is not announcing the descriptive information on the Microsoft logo image in focus mode. Hence the users will not know the purpose of this image
NVDA is not announcing the descriptive information on the Microsoft logo image
NVDA is announcing as graphic

### Expected behavior:
NVDA should announce the descriptive information on the Microsoft logo image
NVDA should announce as 'Microsoft logo graphic'

#### NVDA version:
NVDA version: 2022.1

#### Windows version:
Window 11 version 21H2 (OS Build 22000.795)

#### Does the issue still occur after restarting your computer?
yes 

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No 

#### If NVDA add-ons are disabled, is your problem still occurring?
No

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.

- You have not mentioned what application you are using to reproduce this bug. We assume this being reported as a bug with Outlook. Can you please provide the version of Outlook you are using.
- Can this be reproduced with a plain HTML sample email in a web browser?

Kind Regards,
NV Access Software Developers

**Comment by feerrenrut** [comment]
Opening this template with Outlook, and using Accessibility Insights, the text "Microsoft footer logo" is exposed in the "name" and help text fields.

**Comment by Adriani90** [comment]
Closing as duplicate of #14217 which contains more details.
